bot/cogs/gif.py

```diff
@@ -0,0 +1,32 @@
+from os import environ
+
+import aiohttp
+from discord.ext import commands
+
+
+class SpookyGif:
+    """
+    A cog to fetch a random spooky gif from the web!
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.GIPHY_TOKEN = environ.get('GIPHY_TOKEN')
+
+    @commands.command()
+    async def gif(self, ctx):
```

Comment:
`.spookygif` would probably be a more appropriate command, `.gif` will be fine as an alias

Reply:
.gif as an alias sorta conflicts with our new idea of how this bot should work. doing `.sgif` instead.